Resources/Prototypes/_Floof/Entities/Clothing/Head/misc.yml

```
@@ -41,3 +41,96 @@
     sprite: _Floof/Clothing/Head/hoodmarshoodie.rsi
   - type: Clothing
     sprite: _Floof/Clothing/Head/hoodmarshoodie.rsi
+
+- type: entity
+  parent: ClothingHeadEVAHelmetBase
+  id: ClothingHeadHelmetPilot
+  name: Freelancer "Pilot" Helmet
+  description: A robust helmet designed for extravehicular activites. This version has a issue with the visor flashing from time to time.
+  components:
+  - type: BreathMask
+  - type: Sprite
+    sprite: _Floof/Clothing/Head/pilothelmet.rsi
+  - type: Clothing
+    sprite: _Floof/Clothing/Head/pilothelmet.rsi
+  - type: Tag
+    tags:
+    - HelmetEVA
+    - WhitelistChameleon
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Heat: 0.9
+  - type: HideLayerClothing
+    slots:
+    - Hair
+    - Snout
+    - HeadSide
+    - HeadTop
+  - type: IngestionBlocker
+  - type: IdentityBlocker
```

Some of these components are defined in the parent and are not neccessary here.
```suggestion
  - type: Sprite
    sprite: _Floof/Clothing/Head/pilothelmet.rsi
  - type: Clothing
    sprite: _Floof/Clothing/Head/pilothelmet.rsi
  - type: Tag
    tags:
    - HelmetEVA
    - WhitelistChameleon
  - type: Armor
    modifiers:
      coefficients:
        Blunt: 0.9
        Slash: 0.9
        Piercing: 0.9
        Heat: 0.9
```

---

Stremlined as requested in newest update.